pyproject.toml

```diff
@@ -144,13 +148,88 @@ testpaths = [
 [tool.ruff]
 include = ["prymer/**/*.py", "tests/**/*.py"]
 line-length    = 100
-target-version = "py311"
+target-version = "py312"
```

Comment:
Looks like the project is now Py3.12+?

Reply:
For now.  I'd like to re-enable 3.11 when we're doing making big changes.

Reply:
It doesn't make a difference right now so I reverted to `py311`.